modules/identity/src/Volo.Abp.Identity.HttpApi/Volo/Abp/Identity/ProfileController.cs

```diff
@@ -7,6 +7,7 @@ namespace Volo.Abp.Identity
     [RemoteService]
     [Area("identity")]
     [ControllerName("Profile")]
+    [Route("/api/identity/profiles")]
```

Comment:
`/api/identity/my-profile` would be better since this service is not managing profiles, but only manages it for the current user.